guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -94,8 +94,8 @@ Remote Execution result upload
 | 443 | TCP | HTTPS | Amazon EC2, Azure, Google GCE | Compute resources | Virtual machine interactions (query/create/destroy) (optional)
 ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
-| 443 | TCP | HTTPS | cloud.redhat.com | Red{nbsp}Hat Cloud plugin API calls |
-| 443 | TCP | HTTPS | Red{nbsp}Hat Portal | SOS report | Assisting support cases (optional)
+| 443 | TCP | HTTPS | Red{nbsp}Hat Hybrid Cloud Console | Red{nbsp}Hat Cloud plugin API calls |
+| 443 | TCP | HTTPS | Red{nbsp}Hat Customer Portal | SOS report | Assisting support cases (optional)
 | 443 | TCP | HTTPS | Red{nbsp}Hat CDN | Content Sync | Red{nbsp}Hat CDN
```

Comment:
FYI: there's an article on the IPs for the CDN: https://access.redhat.com/articles/1525183.

Reply:
It's mentioned in another module.